Modules/User/Resources/views/user-settings/index.blade.php

```diff
@@ -0,0 +1,28 @@
+@extends('user::layouts.master')
+@section('content')
+
+<form action ="{{ route('user.settings-hr') }}" method = "POST">
+@csrf
+<div class="container">
+    <div class="col-6">
+        <label class="user-settings">User Settings</label>
+        <input type="text" class="form-control search" placeholder="Search Settings">
+    </div>
+    <div class="card-header d33">
+        <ul class="nav nav-tabs card-header-tabs">
+            <li class="nav-item">
+                <a class="nav-link" href="/user/user-settings">HR</a>  
+            </li>
+            <li class="nav-item">
+                <a class="nav-link active" aria-current="true" href="#">Finance</a>     
```

Comment:
since there are no settings for finance right now, can we name it as:
```suggestion
                <a class="nav-link active" aria-current="true" href="#">Finance (coming soon)</a>     
```